PureBasicIDE/ColorPicker.pb

```diff
@@ -1114,6 +1114,19 @@ Procedure ColorPicker_Name_Event(*Entry.ColorPickerData, Gadget, Type)
       EndIf
       
     Case #GADGET_Color_Canvas1
+
+      If Type = #PB_EventType_MouseWheel
+        *Entry\First-GetGadgetAttribute(#GADGET_Color_Canvas1,#PB_Canvas_WheelDelta)
```

Comment:
Please add spaces between individual expression parts and between procedure parameters to improve readability. I.e.:
```
If *Entry\First > *Palette\Count - GetGadgetAttribute(#GADGET_Color_Scroll, #PB_ScrollBar_PageLength)
```

Reply:
Right :-) , I'm trying to get the hang of this Github stuff ;-)